### DISCUSSION TITLE
About "colour.RGB_to_RGB" definition chromatic adaptation default.

### DISCUSSION THREAD

**Discussion by scoopxyz** [colour-science/colour#700] [Ideas]
[source](https://github.com/colour-science/colour/blob/master/colour/models/rgb/rgb_colourspace.py#L503-L506)

This is a bit of a null issue, as I can already see both sides of the argument, but I just wanted to prompt a discussion around defaulting to using white point adaptation when creating RGB to RGB matrices, and whether people prefer the _opt-out_ rather than _opt-in_ model.

**Comment by thomasmansencal** [comment]
Hi Sean,

I think in the usage of the API so far I have more instances where I want to perform the adaptation than instances where I don't. What about you?

**Comment by KevinJW** [comment]
Mine are in the order CAT02 (colour science, ACES, most camera vendors), Bradford (ICC/Photoshop, Red), XYZ scaling only (Baselight), None (replication of something else that is usually  doing it wrong :-)

other CATs are for matching historical reasons or in specific situations.

Kevin

**Comment by scoopxyz** [comment]
Yeah, I don't know why but it just struck me as odd when I noticed it was there. I've been playing around with using alternate white points for DCI P3 and ACES AP1 incarnations, which I can clearly  accomplish via abother route, but just wanted to see if I was the only one who wasn't expecting it to be there. Seems that way :)

**Comment by nick-shaw** [comment]
I would say my order is the same as Kevin's. The only time I use no CAT is for matrices for ACES "D60 sim" ODTs.

**Comment by scoopxyz** [comment]
Thoroughly convinced with the present implementation :) Is there an ongoing forum of conversation outside of _Issues_ ?

**Comment by thomasmansencal** [comment]
@scoopxyz: Indeed! I'll ping you offline!
